Source/Scene/Camera.js

```diff
@@ -42,9 +42,9 @@ import SceneMode from "./SceneMode.js";
  *
  * An orientation given by numeric heading, pitch, and roll
  *
- * @property {number} heading The heading in radians
- * @property {number} pitch The pitch in radians
- * @property {number} roll The roll in meters
+ * @property {number} [heading] The heading in radians
+ * @property {number} [pitch] The pitch in radians
+ * @property {number} [roll] The roll in meters
```

Comment:
Good catch that these should be optional. If a property is optional, we should document what the defaults are. 

```suggestion
 * @property {number} [heading=0.0] The heading in radians
 * @property {number} [pitch=-CesiumMath.PI_OVER_TWO] The pitch in radians
 * @property {number} [roll=0.0] The roll in meters
```